src/fhirdefs/load.ts

```diff
@@ -203,8 +203,13 @@ export function cleanCachedPackage(packageDirectory: string): void {
  * Loads custom resources defined in resourceDir into FHIRDefs
  * @param {string} resourceDir - The path to the directory containing the resource subdirs
  * @param {FHIRDefinitions} defs - The FHIRDefinitions object to load definitions into
+ * @param {string[]} pathResourcePaths - optional, an array of paths to additional directories containing predefined resources
  */
-export function loadCustomResources(resourceDir: string, defs: FHIRDefinitions): void {
+export function loadCustomResources(
+  resourceDir: string,
+  defs: FHIRDefinitions,
+  pathResourcePaths: string[] = null
+): void {
```

Comment:
I feel like it makes more sense to put this new parameter directly after the `resourceDir` parameter -- because it is very similar.  That might mean having to change a few more places where this function is called, but I assume that's not very many places anyway.  I also think that `additionalResourcePaths` might be a better name.

Reply:
I actually moved all of the `config.paramater` parsing into `load.ts` so we could get that extra bit of code coverage. Let me know what you think of the new parameter names and ordering.